packages/cli/src/index.ts

```diff
@@ -21,6 +20,23 @@ async function init() {
   await program.parseAsync(process.argv)
 }
 
+const events = ["exit", "SIGINT", "SIGUSR1", "SIGUSR2", "uncaughtException"]
+events.forEach(event => {
+  process.on(event, (evt?: number) => {
+    if (evt && !isNaN(evt)) {
+      return
+    }
+    if (evt) {
+      console.error(
+        error(
+          "Failed to run CLI command - please report with the following message:"
+        )
+      )
+      console.error(error(evt))
+    }
+  })
+})
```

Comment:
wot dis

Reply:
Moving something that was done as part of the prebuild code to handle errors up to the top level, as the prebuilds step has been removed.